src/utils/string/index.ts

```diff
@@ -32,3 +33,36 @@ export const omitTypename = (object) =>
 const DATE_FORMAT = "MMM d, yyyy, h:mm:ss aaaa";
 export const getDateCopy = (d: Date): string =>
   d ? format(new Date(d), DATE_FORMAT) : "";
+
+export const copyToClipboard = (str: string) => {
```

Comment:
Can we use something like react-copy-to-clipboard instead of implementing this ourselves?

Reply:
i disagree i dont think adding an extra package thats 37kb large is worth the extra bloat when we can accomplish the same functionality in 7 lines of code.